File: Microsoft Office Developer Documentation Team/SharePoint-2010-Creating-a-NET-Connectivity-4143/[VB]-SharePoint 2010 Creating a .NET Connectivity Assembly for BCS/VB/BDCModel/BdcModel1/TrainingEvent.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

''' <summary>
''' This class contains the properties for Entity1. The properties keep the data for Entity1.
''' If you want to rename the class, don't forget to rename the entity in the model xml as well.
''' </summary>
Partial Public Class TrainingEvent
    Private _TrainingEventID As Int32
    Private _EventDate As DateTime
    Private _Status As String
    Private _LoginName As String
    Private _Title As String
    Private _EventType As String
    Private _Description As String
    Public Property TrainingEventID() As Int32
        Get
            Return _TrainingEventID
        End Get
        Set(ByVal value As Int32)
            _TrainingEventID = value
        End Set
    End Property

    Public Property EventDate() As DateTime
        Get
            Return _EventDate
        End Get
        Set(ByVal value As DateTime)
            _EventDate = value
        End Set
    End Property
    Public Property Status() As String
        Get
            Return _Status
        End Get
        Set(ByVal value As String)
            _Status = value
        End Set
    End Property
    Public Property LoginName() As String
        Get
            Return _LoginName
        End Get
        Set(ByVal value As String)
            _LoginName = value
        End Set
    End Property
    Public Property Title() As String
        Get
            Return _Title
        End Get
        Set(ByVal value As String)
            _Title = value
        End Set
    End Property
    Public Property EventType() As String
        Get
            Return _EventType
        End Get
        Set(ByVal value As String)
            _EventType = value
        End Set
    End Property
    Public Property Description() As String
        Get
            Return _Description
        End Get
        Set(ByVal value As String)
            _Description = value
        End Set
    End Property

End Class

File: Microsoft Office Developer Documentation Team/SharePoint-2010-Creating-a-NET-Connectivity-4143/[C#]-SharePoint 2010 Creating a .NET Connectivity Assembly for BCS/C#/BDCModel/BdcModel1/TrainingEvent.cs
    using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace BDCModel.BdcModel1
{
    /// <summary>
    /// This class contains the properties for Entity1. The properties keep the data for Entity1.
    /// If you want to rename the class, don't forget to rename the entity in the model xml as well.
    /// </summary>
    public partial class TrainingEvent
    {
        //TODO: Implement additional properties here. The property Message is just a sample how a property could look like.
        public Int32 TrainingEventID { get; set; }
        public DateTime EventDate { get; set; }
        public string Status { get; set; }
        public string LoginName { get; set; }
        public string Title { get; set; }
        public string EventType { get; set; }
        public string Description { get; set; } 

    }
}
